zookeeper-server/src/main/java/org/apache/zookeeper/server/Request.java

```diff
@@ -467,26 +467,7 @@ public void logLatency(SummarySet metric, String key) {
      * session
      */
```

Comment:
also please extend the JavaDoc here, that not all users will be returned, only the users which are belonged to a scheme "visible for auditing". (as far as I can tell only audit logs used this method... maybe we could even rename it to `getUsersForAudit`)

Reply:
I have extended the JavaDoc, but not mentioning any filtering as I have now removed it :)  ([Rationale](https://github.com/apache/zookeeper/pull/1519#issuecomment-725535273))